src/editor/components/PageSettings/PageSettings.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../../styles/variables.scss';
+@import '../../../../styles/mixins.scss';
+@import '../../styles/root.scss';
+@import '../../styles/variables.scss';
+@import '../../styles/mixins.scss';
+
+$block: '.#{$ns}editor-page-settings';
+
+#{$block} {
+    &__container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: 100%;
+        padding: $indentXXS;
+        border-bottom: 1px solid var(--g-color-line-generic);
+
+        .g-disclosure__content {
```

Comment:
Because we cannot customize the content wrapper
 https://github.com/gravity-ui/uikit/blob/main/src/components/Disclosure/DisclosureDetails/DisclosureDetails.tsx#L32